common/ux-window.cpp

```diff
@@ -215,13 +216,20 @@ namespace rs2
    
     void ux_window::open_window()
     {
+        IMGUI_CHECKVERSION();
```

Comment:
What will this do?

Reply:
It checks the compatibility of the ImGui library used , I added it because it was recommended on the Imgui repo, I can remove if we want to